artifactory/commands/python/pip.go

```diff
@@ -0,0 +1,80 @@
+package python
+
+import (
+	"io"
+	"os/exec"
+
+	"github.com/jfrog/build-info-go/entities"
+	"github.com/jfrog/build-info-go/utils/pythonutils"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/python/dependencies"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+)
+
+type PipCommand struct {
+	PythonCommand
+}
+
+func NewPipCommand() *PipCommand {
+	return &PipCommand{PythonCommand: PythonCommand{pythonTool: pythonutils.Pip}}
+}
+
+func (pc *PipCommand) Run() (err error) {
+	return pc.PythonCommand.Run()
+}
+
+func (pc *PipCommand) UpdateDepsChecksumInfoFunc(dependenciesMap map[string]entities.Dependency, srcPath string) error {
+	servicesManager, err := utils.CreateServiceManager(pc.serverDetails, -1, 0, false)
+	if err != nil {
+		return err
+	}
+	return dependencies.UpdateDepsChecksumInfo(dependenciesMap, srcPath, servicesManager, pc.repository)
+}
+
+func (pc *PipCommand) SetRepo(repo string) *PipCommand {
+	pc.PythonCommand.SetRepo(repo)
+	return pc
+}
+
+func (pc *PipCommand) SetArgs(arguments []string) *PipCommand {
+	pc.PythonCommand.SetArgs(arguments)
+	return pc
+}
+
+func (pc *PipCommand) SetCommandName(commandName string) *PipCommand {
+	pc.PythonCommand.SetCommandName(commandName)
+	return pc
+}
+
+func (pc *PipCommand) CommandName() string {
+	return "rt_python_pip_command"
```

Comment:
rt_python_pip_command --> rt_python_pip

The `_command` is redundant. The same goes for `rt_python_pipenv_command` and `rt_python_poetry_command`.